### PR TITLE
#78-Resolved coding standard issues in modules files

### PR DESCRIPTION
**Pull request by pritamyj**
Resolved coding standard issues and warnings in custom modules files

### CODE REVIEW

File: web/modules/custom/profile_block/src/Controller/Controller.php [hunk 1]
```diff
@@ -24,29 +22,53 @@ class Controller extends ControllerBase {
...
+   * @param \Drupal\Core\Routing\CurrentRouteMatch $currentRouteMatch
+   *   The current route match.
+   * @param \Drupal\Core\Entity\EntityTypeManager $entity_type_manager
+   *   The entity type manager.
    */
-  public function __construct(Connection $connection) {
+  public function __construct(Connection $connection, CurrentRouteMatch $currentRouteMatch, EntityTypeManager $entity_type_manager) {
     $this->connection = $connection;
+    $this->currentRouteMatch = $currentRouteMatch;
+    $this->entity_type_manager = $entity_type_manager;
```
Comment (guptahemant) [inline web/modules/custom/profile_block/src/Controller/Controller.php]: $this->entity_type_manager should be $this->entityTypeManager right

File: web/modules/custom/profile_block/src/Controller/Controller.php [hunk 2]
```diff
@@ -199,13 +208,8 @@ public function followersmodal() {
...
         '#post_uri_all' => $post_uri_all,
         '#link' => 'Remove',
-        '#flag_txt' => t(
-                "<h2>Remove Follower?</h2><p class='remove_txt'>$flag_txt</p>"
-        ),
-        '#flag_link' => Link::fromTextAndUrl(
-                t('Remove'),
-                $remove_link
-        )->toString(),
+        '#flag_txt2' => $this->t("Instagram wont tell") . " " . $uname . " " . $this->t("they were removed from your followers"),
```
Comment (guptahemant) [inline web/modules/custom/profile_block/src/Controller/Controller.php]: Use variable replacement inside t functions here.

File: web/modules/custom/profile_block/src/Controller/CustomModalController.php [hunk 3]
```diff
@@ -25,22 +24,44 @@ class CustomModalController extends ControllerBase {
...
+   * @param \Drupal\Core\Routing\CurrentRouteMatch $currentRouteMatch
+   *   The current route match.
+   * @param \Drupal\Core\Entity\EntityTypeManager $entity_type_manager
+   *   The entity type manager.
    */
-  public function __construct(Connection $connection) {
+  public function __construct(Connection $connection, CurrentRouteMatch $currentRouteMatch, EntityTypeManager $entity_type_manager) {
     $this->connection = $connection;
+    $this->currentRouteMatch = $currentRouteMatch;
+    $this->entity_type_manager = $entity_type_manager;
```
Comment (guptahemant) [inline web/modules/custom/profile_block/src/Controller/CustomModalController.php]: entity_type_manager should be entityTypeManager

File: web/modules/custom/profile_block/src/Controller/CustomModalController.php [hunk 4]
```diff
@@ -257,8 +272,8 @@ public function followingmodal() {
         '#following_count' => $following_count,
         '#post_uri_all' => $post_uri_all,
         '#link' => 'Following',
-        '#flag_txt' => t("<p class='unfollow_txt'>If you change your mind, you'll have to request to follow @$uname again.</p>"),
-        '#flag_link' => Link::fromTextAndUrl(t('Unfollow'), $flag_link)->toString(),
+        '#flag_txt' => $this->t("If you change your mind, you'll have to request to follow @") . $uname . " " . $this->t("again."),
```
Comment (guptahemant) [inline web/modules/custom/profile_block/src/Controller/CustomModalController.php]: Use t method with variable replacement

**Review by guptahemant — CHANGES_REQUESTED** [review]

File: web/modules/custom/profile_block/src/Controller/Controller.php (hunk 1 above)
Comment (pritamyj) [reply web/modules/custom/profile_block/src/Controller/Controller.php]: fixed here: https://github.com/guptahemant/insta-clone/pull/79/commits/33d95104fb8e57192cee781f012affa9093c6567#diff-27ed9e15a36bba030870a9e341b1c19a7d0857dec15759d84cd2029f8965dd24

File: web/modules/custom/profile_block/src/Controller/Controller.php (hunk 2 above)
Comment (pritamyj) [reply web/modules/custom/profile_block/src/Controller/Controller.php]: fixed here: https://github.com/guptahemant/insta-clone/pull/79/commits/33d95104fb8e57192cee781f012affa9093c6567#diff-27ed9e15a36bba030870a9e341b1c19a7d0857dec15759d84cd2029f8965dd24

File: web/modules/custom/profile_block/src/Controller/CustomModalController.php (hunk 3 above)
Comment (pritamyj) [reply web/modules/custom/profile_block/src/Controller/CustomModalController.php]: fixed here: [web/modules/custom/profile_block/src/Controller/CustomModalController.php](https://github.com/guptahemant/insta-clone/pull/79/commits/33d95104fb8e57192cee781f012affa9093c6567#diff-a8c43901147783f211bc5d3e20d8b1634bd35b87941fb640c22e89af8a2765d5)

File: web/modules/custom/profile_block/src/Controller/CustomModalController.php (hunk 4 above)
Comment (pritamyj) [reply web/modules/custom/profile_block/src/Controller/CustomModalController.php]: fixed here: [web/modules/custom/profile_block/src/Controller/CustomModalController.php](https://github.com/guptahemant/insta-clone/pull/79/commits/33d95104fb8e57192cee781f012affa9093c6567#diff-a8c43901147783f211bc5d3e20d8b1634bd35b87941fb640c22e89af8a2765d5)

**Review by VighneshSadagopal — APPROVED** [review]

**Review by guptahemant — APPROVED** [review]

**Pull request merged**